docs/extensions/model-merger.md

```diff
@@ -0,0 +1,66 @@
+# Model Merger
+
+The model merge is used to automatically merge two models. Merge is not meant in the same way as in terms VCS systems like git. It‘s main use case is incrementalisation of importer at the moment but is not limited to it.
+## Model Merge Specification 
+
+The model merge needs a customizable specification of how to merge instances of a given concept. The DSL should support the user to write correct specifications for a set of concepts and allow to reuse specifications for instance when a concept is extended or interfaces are implemented. Putting this into a DSL rather than writing this in plain java allows us to evolve the runtime and the language independently. Inspirations can be taken from the new VCD aspect in MPS. While a much simpler use case it might still be useful input.
+
+### Concrete Syntax Example
+```
+Spec
+----
+<Concept> -> Id : (node) = node.name
+                         ---- Properties ----
+                         name -> left | right | manual (left, right, node) -> <function returning the new property value>
+                        prop 2 -> 
+                        ---- Children ----
+                        childA [0..1] -> left | right | drop | auto | manual (left, right, node) -> <function returning the node used for the child>
+                        child [1] -> left | right | auto | manual 
+                        child [0..n] -> 
+                                               new on the right : add | drop
+                                               only existing on the the left: keep | drop 
+                                               element:  left | right | drop | auto | manual 
+```
```

Comment:
Don't we have yet an example for references?If not I think we can add it do the open questions.

Reply:
Yeah a "Example" section would make sense. I think we can add it to the paragraph about the individual operations.